### DISCUSSION TITLE
More options for `rebuild`

### DISCUSSION THREAD

**Discussion by Stanzilla** [pnpm/pnpm#6466] [Q&A] [answered]
I'm currently using `npm install --platform=darwin --arch=x64 sharp && npm rebuild --platform=darwin --arch=arm64 sharp` in my CI and am wondering how to do the same with pnpm. Just doing `pnpm rebuild sharp` did not have the desired result.

**Comment by zkochan** [comment, marked as answer]
Try

```
pnpm install --config.platform=darwin --config.arch=x64 sharp && pnpm --config.platform=darwin --config.arch=arm64 rebuild sharp
```

Also, what does `npm rebuild sharp` mean?

**Reply by Stanzilla** [reply]
Thanks, that seems to have done the trick! What do you mean with your question? Why I have to rebuild sharp for arm here? I need both the x64 and arm64 binaries.
